File: 1270_shuffle_the_array/Solution.cpp
class Solution {
public:
    vector<int> shuffle(vector<int>& nums, int n) {
        std::vector<int> ret(n*2);
        for(int i = 0; i < n; i++){
            ret[i*2] = nums[i];
            ret[(i*2) + 1] = nums[i + n];
        }
        return ret;
    }
};

File: 1270_shuffle_the_array/Solution.js
/**
 * @param {number[]} nums
 * @param {number} n
 * @return {number[]}
 */
var shuffle = function(nums, n) {
    var ret = [n];
    for(var i = 0; i < n; i++){
        ret[i*2] = nums[i];
        ret[(i*2) + 1] = nums[i + n];
    }
    return ret;
};
